.github/workflows/ci-build-publish-main.yml

```diff
@@ -44,3 +44,22 @@ jobs:
           $DOCKER_TAG_SHA
       env:
         COSIGN_EXPERIMENTAL: "true"
+
+  app_deployment:
+    env:
+      cd_workflow_dispatch_file_name: cd-app-deployment.yml
+      infrastructure_repo_path: buerokratt/Infrastructure
+      app_name: mock-bot
+    runs-on:
+      ubuntu-latest
+    defaults:
+      run:
+        shell : bash
+    needs :
+      PackageDeploy
+    steps:
+      - name: Initiate k8s app deployment
+        run: |
+          gh workflow run $cd_workflow_dispatch_file_name -f app_name=$app_name -f image_tag=${GITHUB_SHA} --repo $infrastructure_repo_path --ref main
+        env:
+          GITHUB_TOKEN: ${{ secrets.MULTI_REPO_WORKFLOW_ACCESS_TOKEN }}
```

Comment:
[https://github.com/buerokratt/Mock-Classifier/pull/50#discussion_r897567568](https://github.com/buerokratt/Mock-Classifier/pull/50#discussion_r897567568)

Reply:
@yahya130 Do you want to make the change @keijoraamat is suggesting and get can get this PR approved and committed.

Reply:
Done, thanks